packages/terra-toolbar/tests/jest/Toolbar.test.jsx

```diff
@@ -35,6 +35,23 @@ describe('Toolbar', () => {
     expect(wrapper.hasClass('align-center'));
   });
 
+  it('should render component with aria-label', () => {
+    const wrapper = shallow(<Toolbar ariaLabel="test" />);
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  it('should render component with aria-labelledby', () => {
+    const wrapper = shallow(
+      <div>
+        <Toolbar ariaLabelledBy="test-id" />
+        <div id="test-id">
+          Test content
+        </div>
+      </div>,
```

Comment:
Closer, but the construction with "test content" is similar to what an `aria-controls` test would look like. A visible label describing the toolbar tools/buttons would typically come before the toolbar, and it's purpose is to describe the type of tools within the toolbar:
```suggestion
      <div>
        <h4 id="test-id">Image Tools</h4>
        <Toolbar ariaLabelledBy="test-id" />
      </div>,
```

Reply:
Not the best visual reference, but an example of a set of toolbars side-by-side, each with their own visible label above each toolbar region (e.g. "Color Mode, "Highlight", "Screen Settings", "Zoom", and "Volume Booster" would all be a visible labels for the toolbar region below each): https://ps.w.org/accessibility-enabler/assets/screenshot-2.png?rev=2023391

Reply:
Updated: https://github.com/cerner/terra-core/pull/3678/commits/aa1daf85b135ceb3179d872c812f095d0b5eb5d6#diff-abc5c772f120a26032f43c2cd33cc35d110d3e2559953ea75eaf4b18bb1abe82

@sycombs While less critical to be descriptive in Jest tests, everywhere else we typically we want to avoid using "Test" or "Example" or "Lorem Ipsum" in our sample content, because even tests are informal documentation and (1) makes it difficult to communicate best practices to consumers, and (2) makes it difficult to use for unscripted testing with screenreaders which should mimic real situations.

Reply:
Did some renaming:
[cd05571](https://github.com/cerner/terra-core/pull/3678/commits/cd055714965430fb0530d05315a699ba7613b73c)